tests/index.test.ts

```diff
@@ -327,15 +194,8 @@ describe("Call Account", function () {
 
         await init();
 
-        const tx = { version: 0, target: "102030" };
-        const buf = SvmCodec.newWasmBuffer(tx);
-        const result = SvmCodec.call("wasm_encode_call", buf);
-        const error = SvmCodec.loadWasmBufferError(result);
-        expect(error).toStrictEqual(
-            "The value of a specific field is invalid (`target`)."
-        );
+        expect(() => encodeCall("102030", new Uint8Array(), new Uint8Array()))
+            .toThrow("The value of a specific field is invalid (`target`).");
 
-        SvmCodec.wasmBufferFree(buf);
```

Comment:
It's a good practice to demonstrate proper usage of the API.
If the `svm_codec.wasm` is a long-lived Wasm instance,
it's important not to leak memory.

Reply:
There are no frees in the test (mock client code) - only in the api implementation itself where memory is allocated. So there is no api to free buffers right now - are you saying we should expose the free buffer method?

Reply:
I've made the free function public and the test code need to change to free buffers returned from the api to the client for educational purposes.

Reply:
opening an issue for this.